elastic/kibana
# [Security Solutions] Add risk tab to the user details page

issue: https://github.com/elastic/kibana/issues/129046

## Summary

Add the risk tab to the user details page

* It extracts Risk Score Over Time and Top Risk Score Contributors to shared component.

<img width="1180" alt="Screenshot 2022-04-19 at 14 58 29" src="https://user-images.githubusercontent.com/1490444/164009853-03d5acee-0e91-480b-a19d-c317df8ebe63.png">

<img width="1440" alt="Screenshot 2022-04-19 at 14 58 36" src="https://user-images.githubusercontent.com/1490444/164009860-8502148c-29cb-4c3c-9db9-ec6c9f5eb0ad.png">


## How to test it
1. Enable feature flag `riskyUsersEnabled`
2. Insert and enable user risk transforms https://github.com/elastic/mechagodzilla/pull/188/files (currently not working)
3. Go to the user page and click on the 'users by risk' tab
4. Click on a user name
5. Click on the 'users by risk' tab
6. Make sure the time range includes dates with user risk data 


****Because the documentation and dashboard are pending I removed links to them and left TODO comments for it.**
<img width="1787" alt="Screenshot 2022-05-02 at 10 11 56" src="https://user-images.githubusercontent.com/1490444/166208974-2390f210-6fb4-498c-a9ab-4d3d3d621ca3.png">
<img width="1776" alt="Screenshot 2022-05-02 at 10 29 14" src="https://user-images.githubusercontent.com/1490444/166208978-cb3f620c-b9e1-43ce-8e47-70e6ac0e4e9e.png">
<img width="458" alt="Screenshot 2022-05-02 at 10 37 43" src="https://user-images.githubusercontent.com/1490444/166208980-ce4ea512-9426-4fb3-a189-f62ed18c9034.png">


### Checklist

Delete any items that are not applicable to this PR.
